### PR TITLE
Trans_finder

### PR DESCRIPTION
**Pull request by kpfrang**
Adds flag `trans_finder` to the EventDisplay script. Writes additional information to the event tree and header. These are required for the deep learning based transient detection with `trans_finder`.

### CODE REVIEW

File: pyV2DL3/script/v2dl3_for_Eventdisplay.py [hunk 1]
```diff
@@ -68,6 +68,12 @@
 at boundary value. tolerance = ratio of absolute differece between boundary and parameter\
 value to boundary",
 )
+@click.option(
+    "--trans_finder",
+    is_flag=True,
+    help="Flag to store extra parameters to events. Required to run deep leraning\
```
Comment (GernotMaier) [inline pyV2DL3/script/v2dl3_for_Eventdisplay.py:74]: typo in `deep leraning`

**Review by GernotMaier — COMMENTED** [review]
Is there a reason why we couldn't simply add those column (Xoff, Yoff) always? File size will be minimal larger. Also all the other additions like the mean_pedvars, this could also be useful for other analysis? 

If this is the case, then I suggest to remove the `trans_finder` boolean and add those values always to the fits output file.

**Comment by kpfrang** [comment]
For me, this would be fine. I tried to separate it before, in case there are reasons to not use it by default. If you give me the okay, I can adjust it accordingly.

**Comment by GernotMaier** [comment]
Yes, please go ahead. This is the better solution than adding a lot of if statements.

File: pyV2DL3/script/v2dl3_for_Eventdisplay.py (hunk 1 above)
Comment (kpfrang) [reply pyV2DL3/script/v2dl3_for_Eventdisplay.py]: removed in 2221f18

**Review by GernotMaier — APPROVED** [review]
All good now.

Could you wait with merging until I find time to update the testing using the CI? The FITS output changed, this is why the test is failing.

**Comment by kpfrang** [comment]
Sure, no rush. Thanks for looking into it.

**Review by GernotMaier — COMMENTED** [review]
One thing I forgot: what if the columns Xoff/Yoff/AltAz do not exist? pyV2DL3/fillEvents is also used by the vegas branch of V2DL3. Will we break something? Or do we need to add a test if these branches exist?

**Comment by kpfrang** [comment]
you are right. I added a simple method to check whether the data exists. I also added a check before adding the pedestal variance to the header.

**Review by GernotMaier — APPROVED** [review]
Great, all good. 

I've updated also the CI testing and the tests are successful.

**Pull request merged**